manifest.json

```diff
@@ -323,6 +323,14 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
+      "directory": "nodejs14.x/cookiecutter-aws-sam-hello-powertools-typescript-nodejs",
```

Comment:
Can you move this information into `manifest-v2.json` file?

Reply:
Hi, I've just done so! I see you have updates to node16.x - should I change this in my structure or is it done when merging?